### ISSUE TITLE
PR: LINDA Reforged #55604 causing issues with cold atmos

### ISSUE THREAD

**Issue by Bram961** [tgstation/tgstation#56449]
## [Round ID]: [155081](https://scrubby.melonmesa.com/round/155081)
(although this happens on most rounds that last more than 30 minutes)
## Reproduction: 
Ever since linda was reworked, whenever a breach happened the area will very rapidly cool down to subzero temperatures, often reaching a temperature somewhere between -100C and absolute zero.

I guess this is because "space is infinitely cold" according to ss13 physics. (realistically space is a great insulator due to being a vacuum)

Why is this bad? with the new system this cold spreads so fast you really cant fix it fast enough with space heaters. By the time you're placing the first one down the cold has spread to adjacent rooms and rooms adjacent to those rooms, this causes large sections of the station to become inhospitable and unpassable due to the firelocks.

This happens almost every round as far as manuel goes.

**Comment by kopoba** [comment]
and gas leaks is speedruning station. Plasma leak spread so fast that you cant do anything to stop it.

**Comment by joookschad** [comment]
Opening a can of water vapour will make like half the map slippery

**Comment by joookschad** [comment]
Also this is "coldmos" is usually a death sentence for lizard players or people who don't manage to find a winter coat 30 minutes in.
It makes the rounds last even shorter than they would because it causes the entire station to become a cold firelock hell that you cannot fix.

**Comment by Pugie** [comment]
1 plasma canister gets opened ina  corner of the station and suddenly everywhere has a plasmaflood

**Comment by Sparkezel** [comment]
That also starts burning everywhere AT ONCE :)

**Comment by LemonInTheDark** [comment]
To begin with, this isn't the place for feedback on prs, that should in theory go on the forum. We use the issue tracker purely to record issues and bugs with the game, and while it isn't the best situation, this is mostly intended behavior.

That said, I do have some things to say on the subject. 
Gas moving and settling out isn't a bug, though it's effect on some rounds is concerning. It's been a very long time since we've had a functioning atmospherics system, and some issues have popped up. Carbons react to cold air slower then they used to. Maps are much more open then they used to be, so gas can spread over very large spaces. Individual areas are larger then they used to be, at least I think so. This means that firelocks going off effect larger areas, and create more churn when opening/closing. 

Admittedly, what we have now equalizes about twice as often as linda used to, so diffs will move faster. I've made some changes to make settling more common as well, see #56081. This was done in the name of more consistent behavior and a cheaper atmospherics system, but it has some downsides.

An aside about flames settling over large areas instantly, this isn't behavior I like, but as I said here #56317 it's the consequence of how we deal with molar deltas rather then pressure or heat ones.

In conclusion, I agree that there's some growing pains here, some systems need some tweaking, however I am unwilling to make atmos significantly slower, or revert to the broken system. Holes/floods should have meaning, but interacting with them could be a bit more fun.

If you have more to say on the topic, please leave it in the code feedback section of the forums.
